docs/develop/excel-async-model.md

```diff
@@ -133,10 +133,20 @@ async function main(context: Excel.RequestContext) {
 }
 ```
 
-## Office Scripts Async API reference documentation
+## Converting legacy async scripts to the current model
 
-[!INCLUDE [Async reference documentation](../includes/async-reference-documentation-link.md)]
+The current API model doesn't use `load`, `sync`, or a `RequestContext`. This makes the scripts much easier to write and maintain. Your best resource for converting old scripts is [Stack Overflow](https://stackoverflow.com/questions/tagged/office-scripts). There, you can ask the community for help with specific scenarios. The following guidance should help outline the general steps you'll need to take.
+
+1. Create a new script and copy the old code into it. Be sure not to include the old `main` method signature, using the current `function main(workbook: ExcelScript.Workbook)` instead.
+
+2. Remove all the `load` and `sync` calls. They are no longer necessary.
+
+3. All properties have been removed. You now access those objects through `get` and `set` methods. For example, instead of setting a cell's fill color through property access like this: `mySheet.getRange("A2:C2").format.fill.color = "blue";`, you'll now use methods like this: `mySheet.getRange("A2:C2").getFormat().getFill().setColor("blue");`
```

Comment:
Do you need to say the obvious of "now convert all your other stuff to use get/set"? You imply that here but don't actually say it specifically. Up to you!

Reply:
I thought about that and since you're thinking that too, I'll add an explicit bit.